cognite_toolkit/_cdf_tk/commands/deploy.py

```diff
@@ -239,11 +245,7 @@ def _deploy_resources(
             for duplicate in duplicated:
                 self.warn(LowSeverityWarning(f"Skipping duplicate {loader.display_name} {duplicate}."))
 
-        nr_of_created = nr_of_changed = nr_of_unchanged = 0
-
-        if force_update:
-            to_update.extend(unchanged)
-            unchanged.clear()
```

Comment:
This does not work anymore as not all resources supports update. Instead, this is checked in the `worker.load_resources`